src/test/java/se/lth/base/server/rest/SearchResourceTest.java

```diff
@@ -0,0 +1,220 @@
+package se.lth.base.server.rest;
+
+import com.google.gson.internal.LinkedTreeMap;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import se.lth.base.server.BaseResourceTest;
+import se.lth.base.server.Config;
+import se.lth.base.server.data.*;
+
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.GenericType;
+import java.sql.Date;
+import java.sql.Timestamp;
+import java.util.List;
+
+public class SearchResourceTest extends BaseResourceTest {
+
+    private final GenericType<List<Drive>> DRIVE_LIST = new GenericType<List<Drive>>(){};
+
+    private final int idStart = 3;
+    private final User SEARCH_TEST_1 = new User(idStart, Role.USER, "SearchTest1", "SearchTest1", "Smith", "+4670207579", "test1@commutecompanion.se", 0, Date.valueOf("1995-01-01"), true, 0, 0, 0);
```

Comment:
Depending on whether this is merged before or after Issue #101 , All of these will fail because the username/email is not a correct email and will not pass the sanitizer in credentials.